api/internal/utils/timedcall_test.go

```diff
@@ -62,3 +63,15 @@ func TestTimedCallSlowWithError(t *testing.T) {
 		t.Fail()
 	}
 }
+
+func TestTimedCallGoroutineLeak(t *testing.T) {
+	beforeGroutine := runtime.NumGoroutine()
+	_ = TimedCall("expect no goroutine leaks", time.Second, func() error {
```

Comment:
why we don't catch the error here? are you ignore for some reasons?
![image](https://github.com/kubernetes-sigs/kustomize/assets/73322208/43be5435-656d-4f59-900d-fb4cb2c17e16)

Reply:
I think the function has no special complex logic. So ignore the error.
I just looked at the other test examples and none of them ignored the error, so i need to change it, thanks for the review.